Logger/FullName.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger;
+public record class FullName
+{
+    private string FirstName { get; set; }
+    private string LastName { get; set; } 
+    private string? MiddleName { get; set; } 
+
+    public FullName(string firstName, string lastName)
+    {
+        FirstName = firstName;
+        LastName = lastName;
+        MiddleName = null;
+    }
+    public FullName(string firstName, string lastName, string? middleName)
+    {

```

Comment:
Should probably include checks for if the other two are null or blank
```suggestion
    {
        ArgumentException.ThrowIfNullOrWhiteSpace(firstName);
        ArgumentException.ThrowIfNullOrWhiteSpace(lastName);
```

Reply:
Thank you, just added it!